### ISSUE TITLE
verify PhET-iO instrumentation

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-quadratics#115]
In today's PhET-iO Zoom call, we discussed the process for testing `brand=phet-io`. The consensus was that the lead designer should verify that the implementation matches what was decided during PhET-iO design. And that verification should be done before passing the sim to QA for PhET-iO dev/RC testing.  

The exact process for doing this needs to be determined, and will require consultation/pairing with someone from the PhET-iO team.  @amanda-phet @samreid let's discuss how to proceed.

PhET-iO API design decisions are documented in https://github.com/phetsims/graphing-quadratics/issues/14.

**Comment by pixelzoom** [comment]
Labeling with high priority, since this is a prerequisite to dev testing.

**Comment by samreid** [comment]
We'll cover this in a meeting on Thursday

**Comment by pixelzoom** [comment]
PhET-iO version has been deferred and will not be published with 1.0, see https://github.com/phetsims/graphing-quadratics/issues/101#issuecomment-442939061.  So this issue can be deferred.

**Comment by pixelzoom** [comment]
Verification of PhET-iO instrumentation has moved over to #112, so closing this issue.
